app/blueprints/template/forms.py

```diff
@@ -1,11 +1,22 @@
 from flask_wtf import FlaskForm
-from wtforms import FileField, HiddenField, StringField
+from govuk_frontend_wtf.wtforms_widgets import GovFileInput, GovSubmitInput, GovTextInput
+from wtforms import FileField, HiddenField, StringField, SubmitField
 from wtforms.validators import DataRequired, Regexp
 
 
-class TemplateUploadForm(FlaskForm):
-    template_name = StringField("Template Name", validators=[DataRequired()])
-    file = FileField("Upload File", validators=[DataRequired()])
+class TemplateCreateForm(FlaskForm):
+    template_name = StringField("Template name", widget=GovTextInput(), validators=[DataRequired()])
+    tasklist_name = StringField(
+        "Task name",
+        widget=GovTextInput(),
+        description="For example, Project management. Applicants will see this in the application",
+        validators=[DataRequired()],
+    )
+    file = FileField(
+        "Upload a file", description="Supports JSON files only", widget=GovFileInput(), validators=[DataRequired()]
+    )
```

Comment:
It might be nice to use the FlaskWTForms `FileField` component here, as this lets us use its additional validators `FileSize` and `FileAllowed`, eg.

```suggestion
   from flask_wtf.file import FileAllowed, FileField, FileSize
   ......
    file = FileField(
        "Upload a file",
        description="Supports JSON files only",
        widget=GovFileInput(),
        validators=[
            DataRequired(),
            FileAllowed(upload_set=["json"], message="Select a file with the extension .json"),
            FileSize(max_size=2 * 1024 * 1024, message="Select a file smaller than 2MB"),
        ],
    )
```

Reply:
Fixed